docs/en/sql-reference/functions/null-functions.md

```diff
@@ -1,5 +1,5 @@
 ---
-slug: /en/sql-reference/functions/functions-for-nulls
+slug: /en/sql-reference/functions/null-functions
```

Comment:
You renamed the file for consistency.

I commend you for this effort. Unfortunately, renaming files only in ClickHouse's documentation only works when the sun, the moon and the stars align.

You'd need to add a redirect in another repo. CI currently fails because of that: https://s3.amazonaws.com/clickhouse-test-reports/63520/4c9cc50781928fe1b06ef1a1cc2c55477cd79141/docs_check/run.log 

See here for my recent, unsuccessful attempt to rename a file here: https://github.com/ClickHouse/clickhouse-docs/pull/2277 (I added a redirect to an existing redirect, perhaps that was why everything broke into pieces).

My suggestion is that you split the file renaming into a different PR to unblock the documentation changes.